ironfish/src/wallet/wallet.ts

```diff
@@ -1387,6 +1387,10 @@ export class Wallet {
     this.onAccountRemoved.emit(account)
   }
 
+  async forceDeletedAccountCleanup(): Promise<void> {
```

Comment:
I'd suggest renaming to `forceCleanupDeletedAccounts` to make the name align a bit more with `cleanupDeletedAccounts`